cli/tests/bundle/fixture02.out

```diff
@@ -3,6 +3,7 @@ const mod = function() {
     const c1 = c;
     class C {
     }
+    const __default = C;
```

Comment:
@lucacasonato these sorts of things you are talking about, right?

@kdy1 we have noticed that quite often variables like this are getting injected in bundles but then never used or called... are we missing some sort of hygiene pass?

Reply:
It's problem of dce pass.
I have a plan to rewrite it using actual basic block graph, but I've been busy these days.